vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -17,24 +17,18 @@ build_ids=$3
 branch_name=imageBump/$new_image_version
 pr_title="VersionBump"
 
-# This function finds the current SIG Image version from pkg/agent/datamodelosimageconfig.go
+# This function finds the current SIG Image version from the input JSON file
 find_current_image_version() {
     filepath=$1
-    while read -r p; do
-        if [[ $p == *"LinuxSIGImageVersion"* ]]; then
-            current_image_version=$(echo $p | awk -F'\"' '{print $2}')
-            echo "Image version is $current_image_version, cut from line $p"
-            break
-        fi
-    done < $filepath
+    current_image_version=$(jq -r .version $filepath)
     echo "Current image version is: ${current_image_version}"
 }
 
 # This function replaces the old image version with the new input image version for all relevant files
 update_image_version() {
     sed -i "s/${current_image_version}/${new_image_version}/g" pkg/agent/bakerapi_test.go
     sed -i "s/${current_image_version}/${new_image_version}/g" pkg/agent/datamodel/sig_config.go
-    sed -i "s/${current_image_version}/${new_image_version}/g" pkg/agent/datamodel/sig_config_test.go
```

Comment:
this was also incorrect. sig_config_test.go has some hardcoded values that still rely on this. this has to be updated or the test would have failed https://github.com/Azure/AgentBaker/blob/e6086d14d13e3acc1c1e87eae03706cb6a56c4bd/pkg/agent/datamodel/sig_config_test.go#L60

again, not caught because automation didnt run. please run automation pipeline when making such changes to validate.